mixitup/js/Department of Commerce_files/style.css

```diff
@@ -0,0 +1,147 @@
+body, button{
```

Comment:
I think the Department of Commerce_files folder should be deleted. It looks like it has a bunch of content from the CDUP site, so nothing should be affected by deleting that file. There's also a bunch of CSS files there that have random number+letter names.